tests/testsuite/progress.rs

```diff
@@ -156,9 +162,28 @@ fn never_progress() {
         .file("src/lib.rs", "")
         .build();
 
+    // should not contain
+    // [DOWNLOADING] [..] crates [..]
+    // [..][DOWNLOADED] 3 crates ([..]) in [..]
+    // [BUILDING] [..] [..]/4: [..]
     p.cargo("check")
-        .with_stderr_does_not_contain("[DOWNLOADING] [..] crates [..]")
-        .with_stderr_does_not_contain("[..][DOWNLOADED] 3 crates ([..]) in [..]")
-        .with_stderr_does_not_contain("[BUILDING] [..] [..]/4: [..]")
+        .with_stderr_data(
```

Comment:
Let's revert this for now.  People are unlikely to see these comments when updating snapshots

Reply:
Revered!